### PR TITLE
[Web] Alpha OSK regressions

### PR DESCRIPTION
**Pull request by jahorton**
Corrects the following regressions within the current alpha:

* Longpress keys on embedded were often not being processed correctly.
* Desktop-based layouts on touch form factors were not changing layer properly.

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/147137:id/kvk/index.html)

**Comment by darcywong00** [comment]
The default sil_euro_latin keyboard and khmer_angkor seem to work OK now.

In the Android app, I'm still having some issues with the sil_cameroon_qwerty keyboard (Afade (Latin) -> Cameroon QWERTY) in the emulator. Can't seem to type in these layers:
* shift layer
* special layer from the "green-red-yellow" button

File: web/source/kmwembedded.ts [hunk 1]
```diff
@@ -554,34 +561,35 @@ namespace com.keyman.osk {
...
 
     // Default handling for external keys.
     // Intentionally not assigning K_TAB or K_ENTER so KMW will pass them back
     // to the mobile apps to handle (insert characters or navigate forms).
     if (code == Codes.keyCodes.K_SPACE) {
-      kbdInterface.output(0, Lelem, ' ');
+      kbdInterface.output(0, target, ' ');
       return true;
     } else if (code == Codes.keyCodes.K_BKSP) {
       kbdInterface.defaultBackspace();
```
Comment (darcywong00) [inline web/source/kmwembedded.ts]: Does `defaultBackspace()` need to be changed too? In the KMW test page, the external keyboard "backspace" doesn't work.

Comment (darcywong00) [reply web/source/kmwembedded.ts]: the backspace issue was only on Linux. It works on Windows

Comment (jahorton) [reply web/source/kmwembedded.ts]: Interesting.  Which browser(s) on Linux?  Might it be related to / a conflict with Keyman for Linux?  (I sometimes have 'interesting' interactions with Keyman Desktop on Windows when testing Web.)

Comment (darcywong00) [reply web/source/kmwembedded.ts]: It was Chrome. I don't think I had Keyman for Linux installed at that point, but I'll check next time I'm back in office.

Comment (darcywong00) [reply web/source/kmwembedded.ts]: Confirmed on another Linux setup (no Keyman for Linux installed).
* external backspace key didn't work.
* special tricolor key on `sil_cameroon_qwerty` doesn't work on the default layer

Same setup with `stable-11.0`
* external backspace key works
* tricolor key on `sil_cameroon_qwerty` works on the default layer

Comment (jahorton) [reply web/source/kmwembedded.ts]: Looks like I can get the same results in macOS Chrome.  So, what's Windows doing differently here...?

Comment (darcywong00) [reply web/source/kmwembedded.ts]: Good. Your latest commits fixed all the above issues

**Pull request merged**